feature-libs/asm/components/asm-customer-360/asm-customer-ui-components/asm-customer-table/asm-customer-table.model.ts

```diff
@@ -1,12 +1,21 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 import { KeyValuePair } from '../../asm-customer-360.model';
 
 export interface CustomerTableColumn {
   text: string;
```

Comment:
If adding `i18nTextKey?: string;` get rid of `text: string;`.

Reply:
it will use either.. for the current implementation (using existing API) it uses i18nTextKey but if the translated string comes from back end then need to use text